### DISCUSSION TITLE
All/some jobs were not successful. PR run failed: Test deployment  - Update version tags ....

### DISCUSSION THREAD

**Discussion by mathematicw** [community/community#75005] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i having those reports coming to my email constantly long time. What to do pls ?
[my git hub name]PR run failed: Test deployment - Update version tags...
[my git hub name]PR run failed: Test deployment, Attempt #2 - - ....
[my git hub name] Run failed: .github/workflows/main.yml - main...

**Comment by ivnvxd** [comment]
If the question is only about the notifications, then to unsubscribe from specific notifications on GitHub:

Open the email notification you received.
Click on the "[Manage your GitHub Actions notifications](https://github.com/settings/notifications)" link.
There you can unsubscribe from that specific type of notification. Look for:
```
Actions
Notifications for workflow runs on repositories set up with GitHub Actions
```

This won't fix the underlying issue causing the failures though, but it will stop the notifications.

**Comment by mathematicw** [comment]
Oh thank you for your prompt reply! I ofc have all those errors which you implied at my git, and almost finally i can't use git on my PC at all.
In short, I definitely changed those service files - main.yaml, package.json, test, yarn, deploy or mb others .. and commited em , and ran `git rebase`and pushed .. got abs mess. What would you recommend? But I ofc dont want to delete that repo and refork it bcz I have PRs made from my GitHUb page. Ofc i'd like to understand all those workflows, but I need to hurry up
Could you pls help ?

Sent with [Proton Mail](https://proton.me/) secure email.

**Reply by ivnvxd** [reply]
There is no simple answer how to fix everything.
Review the error logs from the failed PR runs. They often provide specific details about what went wrong. Examine the changes you made to the service files (main.yaml, package.json, etc.). Look for syntax errors, incorrect configurations, or incompatible dependencies. If you can identify a recent commit where everything was working fine, consider using `git revert` to undo the changes that introduced errors.
